packages/client/hmi-client/src/services/concept.ts

```diff
@@ -55,14 +55,10 @@ async function getFacets(type: AssetType, curies?: ClauseValue[]): Promise<Conce
  * Get DKG entities, either single ones or multiple at a time
  */
 async function getCuriesEntities(curies: Array<string>): Promise<Array<DKG> | null> {
-	try {
-		const response = await API.get(`/mira/currie/${curies.toString()}`);
-		if (response?.status !== 200) return null;
-		return response?.data ?? null;
-	} catch (error) {
-		logger.error(error, { showToast: false });
-		return null;
-	}
+	const response = await API.get(`/mira/currie/${curies.toString()}`);
+	if (response?.status === 200 && response?.data) return response.data;
+	if (response?.status === 204) console.warn('No DKG entities found for curies:', curies);
```

Comment:
The changes made to `api.ts` for error messages will let a 204 slip by. If we want the user to know about this we'll need to pop up a toast too

Reply:
This is in purpose, kind of hiding DKG lack of biological ontologies